src/lib/composables/useSize.ts

```diff
@@ -8,13 +8,13 @@ interface UseSize {
   style?: string;
 }
 
-export function useSize(sizeProp: any): UseSize {
+export function useSize(sizeProp: unknown): UseSize {
```

Comment:
Not sure I agree with this one... The sizeProp should be `any` type, not an `unknown` type imho

Reply:
No, `any` gives an error with our config. This is good, because `unknown` should be preferred over `any`.

Reply:
Although it would be even better to have an explicit type!